Gopkg.lock

```diff
@@ -102,31 +153,54 @@
     "leveldb/opt",
     "leveldb/storage",
     "leveldb/table",
-    "leveldb/util"
+    "leveldb/util",
   ]
+  pruneopts = "UT"
   revision = "c4c61651e9e37fa117f53c5a906d3b63090d8445"
 
 [[projects]]
+  digest = "1:03aa6e485e528acb119fb32901cf99582c380225fc7d5a02758e08b180cb56c3"
   name = "github.com/ugorji/go"
   packages = ["codec"]
+  pruneopts = "UT"
   revision = "b4c50a2b199d93b13dc15e78929cfb23bfdf21ab"
   version = "v1.1.1"
 
 [[projects]]
   branch = "master"
+  digest = "1:e7a1ce56dc2fd57e2e1fe88c984583e8345d24a27068a47b134a2f2fd0e095a2"
   name = "golang.org/x/crypto"
   packages = ["sha3"]
+  pruneopts = "UT"
   revision = "c126467f60eb25f8f27e5a981f32a87e3965053f"
 
 [[projects]]
   branch = "master"
+  digest = "1:d773e525476aefa22ea944a5425a9bfb99819b2e67eeb9b1966454fd57522bbf"
   name = "golang.org/x/sys"
   packages = ["unix"]
+  pruneopts = "UT"
   revision = "ac767d655b305d4e9612f5f6e33120b9176c4ad4"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "a58407f81fac1682566f53f17b8576bcca052a1ed8d9a82d60afd6bcef0d64c9"
+  input-imports = [
+    "github.com/2tvenom/cbor",
+    "github.com/anacrolix/utp",
+    "github.com/ccding/go-stun/stun",
+    "github.com/chzyer/readline",
+    "github.com/jbenet/go-base58",
+    "github.com/pkg/errors",
+    "github.com/satori/go.uuid",
+    "github.com/spf13/pflag",
+    "github.com/stretchr/testify/assert",
+    "github.com/stretchr/testify/mock",
+    "github.com/syndtr/goleveldb/leveldb",
+    "github.com/syndtr/goleveldb/leveldb/comparer",
+    "github.com/syndtr/goleveldb/leveldb/opt",
+    "github.com/ugorji/go/codec",
+    "golang.org/x/crypto/sha3",
+  ]
```

Comment:
@andreyromancev @ita-sammann why so different format? Probably we should fix dep version for consistent packages versioning.

Reply:
@nordicdyno, it's not clear for me. These changes are no more in master and we can't reproduce this behaviour of `dep ensure`.

Reply:
Current dep version is 0.5.0 let's stick to it.